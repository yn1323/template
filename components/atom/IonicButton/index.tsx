import React from 'react'

const IonicButton: React.FC = () => {
  return <ion-button color="primary">Primary</ion-button>
}

export default IonicButton
