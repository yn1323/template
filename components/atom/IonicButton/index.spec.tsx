import React from 'react'
import renderer from 'react-test-renderer'
import IonicButton from '.'

test('AppTitle', () => {
  const component = renderer.create(<IonicButton />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
