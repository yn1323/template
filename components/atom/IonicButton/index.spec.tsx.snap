// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
<ion-button
  color="primary"
>
  Primary
</ion-button>
`;
